content/blog/getting-started-with-hpe-greenlake-api-for-backup-and-recovery.md

```diff
@@ -0,0 +1,591 @@
+The list of the steps to perform this use case using HPE GreenLake API:
+
+1. Use the [API](https://developer.greenlake.hpe.com/docs/greenlake/services/backup-recovery/public/openapi/backup-recovery-public-v1beta1/operation/StoreOncesList/) to discover the StoreOnce instance that can connect to the cloud protection store and copy the id which will be used as the value for storageSystemId as shown in below JSON request body. The API used for this: `GET /backup-recovery/v1beta1/storonces?limit=20&offset=0`
+
+![Discover deployed StoreOnce to create cloud protection store](/img/api-discover-storeonce.png)
+
+2. Next, I discovered the cloud storage at the correct location from the list of the available storage location and copy the `storageLocationId` as shown in below JSON response body. The discovery was done by using a HPE GreenLake API from the data services [set](https://developer.greenlake.hpe.com/docs/greenlake/services/data-services/public/guide/). Note from the below figure that I used filter `“backup-and-recovery”` in capabilities to capture selected storage locations with the correct capability. The below figure shows information about the location (region) where the data will be stored; conversely, it’s located at `“Richmond”`, cloud service provide was `“AZURE”`, and cloud services provider identification is `“eastus2”`. The API used for this: `GET /data-services/v1beta1/storage-locations?filter=”backup-and-recovery” in capabilities`.
+
+![API to figure out the Azure storage location](/img/figure-out-storage-location.png)
+
+3. Moving forward, I composed the cloud protection store at the storage location using values from the previous response body. For this API execution, I created a request JSON body structure for `POST /backup-recovery/v1beta1/protection-stores` that contains the key-pair values from the previous API response. The below figure showed that JSON body to compose the cloud protection store that was connected to the HPE StoreOnce. 
+
+![API composing the protection store](/img/api-compose-protection-store.png)
+
+> The complete information about [this](https://developer.greenlake.hpe.com/docs/greenlake/services/backup-recovery/public/openapi/backup-recovery-public-v1beta1/operation/ProtectionStoreCreate/) JSON body structures is provided on the interactive documentation of `POST /backup-recovery/v1beta1/protection-stores` under Payload tab.
+
+![API Request body JSON to compose protection Store](/img/api-to-create-protection-stores-request-json-body.png)
+
+4. After roughly about 5 minutes, the cloud protection store was completely created based on response of the following `GET /data-services/async-operations` [API](https://developer.greenlake.hpe.com/docs/greenlake/services/data-services/public/openapi/data-services-public-v1beta1/operation/GetAsyncOperation/) as shown in the below figure. 
+
+> ***Note:*** that I used a set of selection parameters in below figure to summarize the information from this task information:
```

Comment:
Rephrase to:
Note: **I used** a set of selection parameters in **the figure below** to provide **a summary of this task information**."